src/libs/actions/Report.js

```diff
@@ -588,13 +588,27 @@ function fetchAllReports(
  * Creates an optimistic chat report with a randomly generated reportID and as much information as we currently have
  *
  * @param {Array} participantList
+ * @param {String} reportName
+ * @param {String} chatType
+ * @param {String} policyID
+ * @param {String} ownerEmail
+ * @param {Boolean} isOwnPolicyExpenseChat
+ * @param {String} oldPolicyName
  * @returns {Object}
  */
-function createOptimisticChatReport(participantList) {
+function createOptimisticChatReport(
+    participantList,
+    reportName = 'Chat Report',
+    chatType = '',
+    policyID = '_FAKE_',
+    ownerEmail = '__FAKE__',
```

Comment:
is this just an existing thing that we do? policyID has one underscore and this has 2

Reply:
oh it must be because i wrote this originally and copy/pasted the response :lolsob: